hail/www/index.md

```diff
@@ -11,15 +11,7 @@ Hail. Post to the [Discussion Forum](http://discuss.hail.is) for user support
 and feature requests. Chat with the Hail team and user community in Hail's
 [Zulip chatroom](https://hail.zulipchat.com).
 
-### Contribute
-
-If you'd like to discuss or contribute to the development of methods or infrastructure, please:
-
-- see the [For Software Developers](https://hail.is/docs/0.2/getting_started_developing.html) section of the installation guide for info on compiling Hail
-- chat with us about development in our [Zulip chatroom](https://hail.zulipchat.com)
-- visit the [Development Forum](http://dev.hail.is) for longer-form discussions
-
-Hail uses a continuous deployment approach to software development, which means we frequently add new features. We update users about changes to Hail via the [Discussion Forum](http://discuss.hail.is). We recommend creating an account on the Discussion Forum so that you can subscribe to these updates as well.
+Hail is actively developed, and new features are frequently added. See the [changelog](https://hail.is/docs/0.2/change_log.html) for more information.
```

Comment:
How about: "Hail is actively developed with new features and performance improvements added weekly."

Reply:
much better.